lib/models/plugins/addCRUDFunctions.js

```diff
@@ -171,6 +171,7 @@ async function getConnection({
 
   const direction = (before) ? BACKWARDS : FORWARDS;
 
+  // TODO: Blocks of code in if and elseif statements are equal.
```

Comment:
That is interesting, @cbishopvelti do you think there is a typo in this code? Looks like the `else if` block below should have set the cursor to `before` instead of `after`.

Reply:
In fact, I'm going to say it's definitely a typo, can you please change that @PrinceWaune and remove this TODO?